app/config/bootstrap.php

```diff
@@ -8,7 +8,8 @@
 // helpers
 require_once HELPERS_DIR.'html_helper.php';
 require_once HELPERS_DIR.'validation_helper.php';
-require_once HELPERS_DIR.'error_helper.php';
+//require_once HELPERS_DIR.'error_helper.php';
```

Comment:
That line will still be used po, I just commented it out po to see the actual error while still debugging, this will redirect the user to not found page po. :smile: